index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Mini Portfolio</title>
+    <link rel="stylesheet" href="./css/style.css">
+    <div class="logo">
```

Comment:
The only thing that should be in the head tag is metadata - not elements you want to actually be on the page.